bpfd/src/command.rs

```diff
@@ -89,6 +89,39 @@ pub(crate) enum Location {
     File(String),
 }
 
+impl Location {
+    pub(crate) async fn get_program_bytes(
+        &self,
+        section_name: &String,
```

Comment:
Instead of passing section_name here just return the section name (if any) from this function, and compare it in the outer `ProgramData.program_bytes()` function